tools/tidy

```diff
@@ -26,7 +26,7 @@ EOF
 set -eo pipefail
 dir="$(dirname "$0")"
 
-args=""
+args="--conf-file $dir/../.tidyallrc"
```

Comment:
What about the other changes the description talks about?

Reply:
Usually I wouldn't repeat the actual code changes in the description, so you can also just drop those. It's more important what we gain from this change. Like running the script from different folders, or via a symlink. I'm not entirely sure yet how I would know I would verify your changes.

Reply:
> Do we need `--profile` or `-pro`?

TL;DR: no.
`--profile` or `-pro` is `perltidy` parameter (https://linux.die.net/man/1/perltidy), but we are using `tidyall` (https://metacpan.org/dist/Code-TidyAll/view/bin/tidyall) which instead uses config. Therefore we just need to make sure it knows it location (by default it expect to have it on system locations or in cwd, that's why there was that cd ..).

Reply:
So something like `cd / && $path-to-os-autoinst-clone/tools/tidy` should work?

Reply:
Well, this obviously requires full path if you are asking for this:

```
$ cd /
$ /home/.../os-autoinst-distri-opensuse/tools/tidy /home/.../os-autoinst-distri-opensuse/tests/kernel/boot_ltp.pm
[checked] tests/kernel/boot_ltp.pm
$ /home/.../os-autoinst-distri-opensuse/tools/tidy
[checked] products/opensuse/main.pm
[checked] products/sle/main.pm
[checked] products/microos/main.pm
[checked] products/alp/main.pm
[checked] products/sle-micro/main.pm
```
But that is required in the current code anyway. And I usually call tidy from git root directory, so that full path is not required.

Reply:
I'm not asking for anything, I'm asking what you want to achieve since you're not explaining it 😃